app/views/modules/_footer.html.erb

```diff
@@ -13,16 +13,55 @@ Unless required by applicable law or agreed to in writing, software distributed
   specific language governing permissions and limitations under the License.
 ---  END LICENSE_HEADER BLOCK  ---
 %>
-   <div class="container-fluid">
-	  <footer>
-	   <ul>
-	   	 <li><%= image_tag "AvalonMediaSystem_Logo_A.png", alt: "Avalon Media System - #{t(:release_label)} #{Avalon::VERSION}" %>
-	     <li><a href="http://www.avalonmediasystem.org">Avalon Media System Project Website</a> </li>
-       <li><%= link_to "Contact Us", comments_path %></li>
-	   </ul>
-	   <div class="release">
-	   	<%=release_text%>
-	   	<%= git_commit_info(" (%s %s [%s])") if Rails.env == 'development' %>
-	   </div>
-       </footer>
+
+<div class="footer dark row-fluid">
+  <div class="container">
+    <div class="row">
+      <div class="col-sm-4">
+        <h2>Quick Links</h2>
+        <ul>
+          <li><a href="/users/sign_in">Login</a></li>
+          <li><a href="/catalog?q=&search_field=all_fields&utf8=%E2%9C%93">Browse</a></li>
+        </ul>
+      </div>
+      <div class="col-sm-4">
+        <h2 class="hidden-xs">Policies</h2>
+        <ul>
+          <li><a href="/about_eraav">About ERA A<span class="plus">&plus;</span>V</a></li>
```

Comment:
Use rails helpers for these instead of hardcoding links.

`link_to 'About ERA A+V', about_path`